### ISSUE TITLE
Can't import xUnit.net xproj => csproj, fails to update the solution

### ISSUE THREAD

**Issue by bradwilson** [dotnet/cli#5345]
## Steps to reproduce

* Clone https://github.com/xunit/xunit (this is on SHA `f400140333e6b15065c16ed9a1b5d0b8c13da69a` which is the current master)
* Update `global.json` SDK version to `1.0.0-rc3-004504`
* Run `dotnet migrate xunit.vs2015.sln`

## Expected  behavior

Migration succeeds.

## Actual behavior

```
  $ dotnet migrate xunit.vs2015.sln
Project xunit.core migration succeeded (C:\Dev\xunit\xunit\src\xunit.core)
Project xunit.assert migration succeeded (C:\Dev\xunit\xunit\src\xunit.assert)
Project xunit.execution.desktop migration succeeded (C:\Dev\xunit\xunit\src\xunit.execution)
Project xunit.console migration succeeded (C:\Dev\xunit\xunit\src\xunit.console)
Project xunit.runner.utility.desktop migration succeeded (C:\Dev\xunit\xunit\src\xunit.runner.utility)
Project xunit.runner.reporters.desktop migration succeeded (C:\Dev\xunit\xunit\src\xunit.runner.reporters)
Project xunit.runner.tdnet migration succeeded (C:\Dev\xunit\xunit\src\xunit.runner.tdnet)
Project xunit.runner.msbuild migration succeeded (C:\Dev\xunit\xunit\src\xunit.runner.msbuild)
Project test.xunit.assert migration succeeded (C:\Dev\xunit\xunit\test\test.xunit.assert)
Project test.utility migration succeeded (C:\Dev\xunit\xunit\test\test.utility)
Project test.xunit.execution migration succeeded (C:\Dev\xunit\xunit\test\test.xunit.execution)
Project test.xunit.console migration succeeded (C:\Dev\xunit\xunit\test\test.xunit.console)
Project test.xunit.runner.msbuild migration succeeded (C:\Dev\xunit\xunit\test\test.xunit.runner.msbuild)
Project test.xunit.runner.reporters migration succeeded (C:\Dev\xunit\xunit\test\test.xunit.runner.reporters)
Project test.xunit.runner.tdnet migration succeeded (C:\Dev\xunit\xunit\test\test.xunit.runner.tdnet)
Project test.xunit.runner.utility migration succeeded (C:\Dev\xunit\xunit\test\test.xunit.runner.utility)
Summary
Total Projects: 16
Succeeded Projects: 16
Failed Projects: 0

Project `C:\Dev\xunit\xunit\src\xunit.execution\xunit.execution.desktop.csproj` does not exist.
.NET Add project(s) to a solution file Command

Usage: dotnet sln <SLN_FILE> add [options] [args]

Arguments:
  <SLN_FILE>  Solution file to operate on. If not specified, the command will search the current directory for one.

Options:
  -h|--help  Show help information

Additional Arguments:
 Add a specified project(s) to the solution.

Project `C:\Dev\xunit\xunit\src\xunit.runner.utility\xunit.runner.utility.desktop.csproj` does not exist.
.NET Add project(s) to a solution file Command

Usage: dotnet sln <SLN_FILE> add [options] [args]

Arguments:
  <SLN_FILE>  Solution file to operate on. If not specified, the command will search the current directory for one.

Options:
  -h|--help  Show help information

Additional Arguments:
 Add a specified project(s) to the solution.

Project `C:\Dev\xunit\xunit\src\xunit.runner.reporters\xunit.runner.reporters.desktop.csproj` does not exist.
.NET Add project(s) to a solution file Command

Usage: dotnet sln <SLN_FILE> add [options] [args]

Arguments:
  <SLN_FILE>  Solution file to operate on. If not specified, the command will search the current directory for one.

Options:
  -h|--help  Show help information

Additional Arguments:
 Add a specified project(s) to the solution.
```

Note that the failed .csproj files names are not correct. We currently use `project.json` to do multi-targeting, and output DLLs with different names per target framework. The .csproj filenames above appear to have the output DLL name, rather than the .csproj name (i.e., it should be `xunit.execution.csproj`, not `xunit.execution.desktop.csproj`).

## Environment data
`dotnet --info` output:

```
.NET Command Line Tools (1.0.0-rc3-004504)

Product Information:
 Version:            1.0.0-rc3-004504
 Commit SHA-1 hash:  b312fbb176

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\1.0.0-rc3-004504
```

**Comment by piotrpMSFT** [comment]
This should be fixed in #5357. @bradwilson can you verify? latest CLI on the repo's home change should have the fix.

**Comment by bradwilson** [comment]
@piotrpMSFT Verified fixed with `1.0.0-rc3-004512`. Thanks!

Now that this works, I'm blocked by dotnet/sdk#665.
